pkgs/build-support/setup-hooks/patch-shebangs.sh

```diff
@@ -29,9 +32,7 @@ patchShebangs() {
         shift
     fi
 
-    local dir="$1"
-
-    header "patching script interpreter paths in $dir"
+    echo "patching script interpreter paths in $@"
```

Comment:
I think header should be kept too

Reply:
But it just calls echo and is marked as obsolete. https://github.com/NixOS/nixpkgs/blob/master/pkgs/stdenv/generic/setup.sh#L115